libs/types/src/tranches.rs

```diff
@@ -1354,7 +1360,7 @@ where
 	}
 }
 
-pub(crate) fn calculate_risk_buffers<Balance, BalanceRatio>(
+pub fn calculate_risk_buffers<Balance, BalanceRatio>(
```

Comment:
This is NOT the location, but I can not make comment on unchanged sections. 

Sections to keep track for invariant: 
* rebalance_tranche
* calculate_price

Reply:
https://github.com/centrifuge/centrifuge-chain/issues/1091